fcrepo-http-api/src/main/java/org/fcrepo/http/api/repository/FedoraRepositoryNodeTypes.java

```diff
@@ -49,6 +49,7 @@
  */
 @Scope("prototype")
 @Path("/fcr:nodetypes")
+@Deprecated
```

Comment:
We should instead add this update to its own PR/ticket... related to:
https://jira.duraspace.org/browse/FCREPO-1890